ci/images/Dockerfile.java17-slim

```diff
@@ -1,5 +1,5 @@
 # Base-image
-FROM registry.suse.com/bci/openjdk-devel:17 AS base
+FROM registry.suse.com/bci/openjdk-devel:117.0.16.0 AS base
```

Comment:
This one breaks the build as well, probably because you are living waaaaaaaay in the future, where Java 117 exists! :stuck_out_tongue_winking_eye:

Reply:
fixed